source/components/resources/rsdumpinfo.c

```diff
@@ -374,6 +374,17 @@ ACPI_RSDUMP_INFO        AcpiRsDumpPinFunction[10] =
     {ACPI_RSD_SHORTLISTX,ACPI_RSD_OFFSET (PinFunction.VendorData),          "VendorData",               NULL},
 };
 
+ACPI_RSDUMP_INFO        AcpiRsDumpClockInput[7] =
```

Comment:
So, it seems in some cases it goes in between of Pin*() in some at the end of the list. Wondering why so...